lib/build_manager.py

```diff
@@ -47,6 +49,48 @@ def __call__(self):
 
         self.build()
 
+    def _build_packages(self, distro, packages):
+        """
+        Build packages
+
+        Args:
+            distro (Distribution): Linux distribution
+            packages ([str]): packages names
+        """
+
+        # create package builder based on distro
+        if distro.lsb_name == "CentOS":
+            mock_config_file = CONF.get('build_packages').get('mock_config').get(
+                distro.lsb_name).get(distro.version)
+            package_builder = mockbuilder.Mock(mock_config_file)
```

Comment:
I don't know if we should keep this distro specifics here, but this looks like a minor drawback from having the build functions removed from the distro class.